packages/blocks/src/api/serializer.js

```diff
@@ -255,33 +261,32 @@ export function getCommentDelimitedContent( rawBlockName, attributes, content )
  * Returns the content of a block, including comment delimiters, determining
  * serialized attributes and content form from the current state of the block.
  *
- * @param {Object} block   Block instance.
- * @param {Object} options Serialization options.
+ * @param {Object}                      block   Block instance.
+ * @param {WPBlockSerializationOptions} options Serialization options.
  *
  * @return {string} Serialized block.
  */
-export function serializeBlock( block, { isNested = false } = {} ) {
+export function serializeBlock( block, { isInnerBlocks = false } = {} ) {
```

Comment:
Should be singular `isInnerBlock`?

**Edit:** Oh, I guess in the context that this is an option just passed along from `serialize` which is for multiple blocks, it makes sense as written, unless we wanted to rename the option as it applies to the context of the singular block being serialized. I think that comes with its own confusion. I'll leave it to you to judge, but I'm fine either way.

Reply:
I think I'd leave it as is.